lib/order.rb

```diff
@@ -0,0 +1,29 @@
+class Order
+
+  attr_reader :current_order, :bill
+
+  def initialize
+    @current_order = {}
+  end
+
+  def take_order(dish, qty)
+    first(dish) ? addmore(dish, qty) : addfirst(dish, qty)
+  end
+
+  private
+
+  def first(dish)
+    @current_order.has_key?(dish)
```

Comment:
Style/DeprecatedHashMethods: `Hash#has_key?` is deprecated in favor of `Hash#key?`. (https://github.com/bbatsov/ruby-style-guide#hash-key)